docs/GraFx-Studio/guides/layouts/index.md

```diff
@@ -61,3 +71,150 @@ From left to right:
 - Rotation
 - Scaling
 
+## Reference Available Layout Preset Sizes
```

Comment:
I didn't review the full list of presets, but I assume you based it on what is being implemented or on the internal documentation.